mobile-app/app/screens/AppNavigator/screens/Loans/screens/AddOrRemoveCollateralScreen.tsx

```diff
@@ -0,0 +1,803 @@
+              ),
+              testID: "add_remove_collateral_vault_share",
+              themedProps: lhsThemedProps,
+            }}
+            rhs={{
+              value: collateralVaultShare.isNaN()
+                ? 0
+                : collateralVaultShare.toFixed(2),
+              testID: "add_remove_collateral_vault_share",
+              suffix: "%",
+              themedProps: rhsThemedProps,
+            }}
+          />
+          <NumberRowV2
+            info={{
+              title: "Available Loan",
+              message:
+                "This is the current loan amount available for this vault.",
+              iconStyle: {
+                light: tailwind("text-mono-light-v2-500"),
+                dark: tailwind("text-mono-dark-v2-500"),
+              },
+            }}
+            containerStyle={{
+              style: tailwind("flex-row items-start w-full mt-5"),
+            }}
+            lhs={{
+              value: translate(
+                "screens/AddOrRemoveCollateralScreen",
+                "Available loan"
+              ),
+              testID: "add_remove_collateral_loan",
+              themedProps: lhsThemedProps,
+            }}
+            rhs={{
+              value: getPrecisedCurrencyValue(vault.loanValue),
+              testID: "add_remove_collateral_loan",
+              prefix: "$",
+              themedProps: rhsThemedProps,
+            }}
+          />
+          <TotalTokenCollateralRow
+            totalTokenBalance={totalTokenBalance}
+            totalUsdAmount={totalTokenValueInUSD}
+            symbol={selectedCollateralItem.token.displaySymbol}
+            collateralFactor={selectedCollateralItem.factor}
+          />
+          <View style={tailwind("pt-5")}>
+            <CollateralizationRatioDisplayV2
+              collateralizationRatio={resultingColRatio.toFixed(2)}
+              minCollateralizationRatio={vault.loanScheme.minColRatio}
+              totalLoanAmount={vault.loanValue}
+              testID="add_remove_collateral"
+              collateralValue={totalVaultCollateralValue.toFixed(8)}
+              showProgressBar
+            />
+          </View>
+        </ThemedViewV2>
+      </ThemedViewV2>
+
+      <View style={tailwind("pt-16 px-7")}>
+        {!disableSubmitButton && (
+          <ThemedTextV2
+            light={tailwind("text-mono-light-v2-500")}
+            dark={tailwind("text-mono-dark-v2-500")}
+            style={tailwind("text-center text-xs font-normal-v2")}
+          >
+            {translate(
+              "screens/AddOrRemoveCollateralScreen",
+              isConversionRequired
+                ? "By continuing, the required amount of DFI will be converted"
+                : "Review full details in the next screen"
+            )}
+          </ThemedTextV2>
+        )}
+
+        <ButtonV2
+          fillType="fill"
+          label={translate("components/Button", "Continue")}
+          disabled={disableSubmitButton}
```

Comment:
Disable btn when resulting collateral ratio is less than min collateral ratio
<img width="378" alt="image" src="https://user-images.githubusercontent.com/53080940/195766029-457fbf9b-4e2c-414b-b2a1-86d6977ce2d9.png">

Reply:
Wrong error message, 
Steps to reproduce 
- add only DUSD as collateral 
-  took any loan
- try to remove max collateral

<img width="378" alt="image" src="https://user-images.githubusercontent.com/53080940/195766473-37bd815a-6926-4057-b720-7bf6e11720f4.png">